api/file.js

```diff
@@ -14,16 +14,20 @@ function readFile (req, res, next) {
   const username = req.session.user.username
   const tutorial = parseInt(req.body.tutorial)
   if (isNaN(tutorial)) {
-    res.status(403).json({status: 'error', message: 'Unrecognizable tutorial number'})
+    res.status(403).json({
+      status: 'error',
+      message: 'Unrecognizable tutorial number'})
     return
   }
+
   fileDB.getFileForUser(username, tutorial)
     .then((result) => {
       res.status(200).json({
         status: 'success',
         fileData: result.data
       })
     })
+    // User retrieves the tutorial for the first time
```

Comment:
Not sure what this comment is referring to?

Reply:
I accidentally left this comment when I rewrote the code.